src/main/kotlin/io/openfuture/state/domain/TransactionRequest.kt

```diff
@@ -0,0 +1,31 @@
+package io.openfuture.state.domain
+
+import io.openfuture.state.model.BlockchainType
+import org.web3j.protocol.core.methods.response.EthBlock
+import kotlin.math.pow
+
+class TransactionRequest(
```

Comment:
It's impossible to understand what the Transaction Request is.

rename to TransactionDto / AddTransactionRequest